### DISCUSSION TITLE
Este osu por qué no tiene una página web para que se haga más conocido?

### DISCUSSION THREAD

**Discussion by RetrogisusDEV** [ppy/osu#16930] [Q&A] [answered]
No quieren una web oficial...
O simplemente van a vivir en sombras...
O tienen una web oculta?
Por que después de descargar 5 tipos de osu! Conseguí este en una página pirata xD

**Comment by frenzibyte** [comment]
Please post discussions in English, I don't understand what you're saying.

**Comment by Walavouchey** [comment, marked as answer]
The lazer version of osu! will be put up on the download page of the website eventually. See https://github.com/ppy/osu-web/issues/8246.

**Comment by MishterKirby** [comment]
La versión lazer de osu! se pondrá en la página de descargas en la página web oficial eventualmente. Ver ppy/osu-web#8246.
